deploy/script/src/config.js

```diff
@@ -0,0 +1,38 @@
+const path = require('path');
+const rootDir = path.resolve(__dirname, './../../../');
+
+const deployConfig =  {
+  api: { dockerRepo: 'paralect/ship-api', dir: `${rootDir}/api` },
```

Comment:
'paralect/ship-api' should this also move to variables?

Reply:
I'm not sure what you mean :( Could you explain?